components/camera/client.go

```diff
@@ -228,6 +228,10 @@ func (c *client) Stream(
 	return stream, nil
 }
 
+func (c *client) GetImage(ctx context.Context) (image.Image, func(), error) {
+	return c.Read(ctx)
+}
+
```

Comment:
This doesn't go over the wire now, but in the next pr we should make this function the function that the client and server uses.

Reply:
My mistake it does.